sed/core/user_dirs.py

```diff
@@ -0,0 +1,45 @@
+"""
+This module sets up the user-specific directory paths for configuration, data, and logs
+using the `platformdirs` library. The directories are created if they do not already exist.
+
+Attributes:
+    USER_CONFIG_PATH (pathlib.Path): The path to the user-specific configuration directory.
+    USER_LOG_PATH (pathlib.Path): The path to the user-specific log directory.
+    USER_DATA_PATH (pathlib.Path): The path to the user-specific data directory.
+"""
+from __future__ import annotations
+
+from pathlib import Path
+
+from platformdirs import user_config_path
+from platformdirs import user_data_path
+from platformdirs import user_log_path
+
+USER_CONFIG_PATH = user_config_path(appname="sed", appauthor="OpenCOMPES", ensure_exists=True)
+USER_LOG_PATH = user_log_path(appname="sed", appauthor="OpenCOMPES", ensure_exists=True)
+USER_DATA_PATH = user_data_path(appname="sed", appauthor="OpenCOMPES", ensure_exists=True)
```

Comment:
How can this be configured?

Reply:
platformdirs basically gives you the standard paths defined by the OS where config, logs and data should go. By giving the appname (we could name it sed-processor) and appauthor, it creates these directories for us and we just have to use it for our application.
These don't need to be configured. Or maybe I misunderstood your question.

Reply:
I think it's not a good idea to put large amounts of data into a folder the user does not have any control over. This will lead to overflowing root drives, huge amounts of data being synchronized, etc.

Reply:
I understand your point. Storing to user data will not be default behavior then.